pkg/controller/statusmanager/status_manager.go

```diff
@@ -346,8 +366,8 @@ func (status *StatusManager) SetFromPods() {
 	// If all pods are being created, mark deployment as not failing
 	status.SetNotFailing(PodDeployment)
 
-	config := &opv1alpha1.NetworkAddonsConfig{}
-	if err := status.client.Get(context.TODO(), types.NamespacedName{Name: names.OPERATOR_CONFIG}, config); err != nil {
+	config := &opv1alpha1.NetworkAddonsConfig{ObjectMeta: metav1.ObjectMeta{Name: status.name}}
+	if err := status.client.Get(context.TODO(), types.NamespacedName{Name: status.name}, config); err != nil {
```

Comment:
why is this change needed?

Reply:
I don't think there is much difference, I just wanted it be consistent with [this](https://github.com/kubevirt/cluster-network-addons-operator/blob/547ad444cd072db8493ce93c37809937904e83f8/pkg/controller/statusmanager/status_manager.go#L97) Get command. since I am using this `config` object I thought I mught as welll changed also the `Get` , and make it like the other `Get`s in this module. but if you think this is not related  here I can turn it back.

Reply:
no I don't mind a little piggy-banking, just wanted to make sure I'm not missing something of essence.